gum/gummodulemap.h

```diff
@@ -27,6 +27,8 @@ GUM_API const GumModuleDetails * gum_module_map_find (GumModuleMap * self,
 
 GUM_API void gum_module_map_update (GumModuleMap * self);
 
+GUM_API const GArray * gum_module_map_get_values (GumModuleMap * self);
```

Comment:
Even though it's the most correct, the `const` part here should be omitted, so the GArray API can be used without casting. E.g. `g_array_get_element_size`, which expects a non-const pointer.